packages/hdwallet-core/src/fio.ts

```diff
@@ -72,6 +72,14 @@ export interface FioWallet extends FioWalletInfo {
   _supportsFio: boolean;
   fioGetAddress(msg: FioGetAddress): Promise<string>;
   fioSignTx(msg: FioSignTx): Promise<FioSignedTx>;
+  fioDecryptRequestContent(msg: FioEncryptionContent): Promise<string>
+  fioEncryptRequestContent(msg: FioEncryptionContent): Promise<string>
+}
+
+export interface FioEncryptionContent {
```

Comment:
FioRequestContent? totally nitpicking, trying to think what makes it clear what this type is for when it pops up in some engineer's typeahead.